Neos.ContentRepository.Core/Classes/DimensionSpace/InterDimensionalVariationGraph.php

```diff
@@ -93,10 +93,9 @@ public function getDimensionSpacePoints(): DimensionSpacePointSet
      */
     public function getWeightedDimensionSpacePoints(): array
     {
-        if (is_null($this->weightedDimensionSpacePoints)) {
+        if (!isset($this->weightedDimensionSpacePoints)) {
             $this->initializeWeightedDimensionSpacePoints();
         }
-        /** @var array<string,WeightedDimensionSpacePoint> $weighedDimensionSpacePoints */
         $weighedDimensionSpacePoints = $this->weightedDimensionSpacePoints;
 
         return $weighedDimensionSpacePoints;
```

Comment:
The local variable is totally useless (and contains a typo)
```suggestion
        return $this->weightedDimensionSpacePoints;
```